edge_computing/ztp-reference-cluster-configuration-for-vdu.adoc

```diff
@@ -30,6 +30,15 @@ include::modules/ztp-enabling-workload-partitioning-sno.adoc[leveloffset=+1]
 
 * For the recommended {sno} workload partitioning configuration, see xref:../edge_computing/ztp-reference-cluster-configuration-for-vdu.adoc#ztp-sno-du-enabling-workload-partitioning_sno-configure-for-vdu[Workload partitioning].
 
+include::modules/ztp-sno-du-disk-encryption.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://docs.openshift.com/container-platform/4.16/security/network_bound_disk_encryption/nbde-about-disk-encryption-technology.html#nbde-tpm-encryption_nbde-implementation[TPM encryption]
```

Comment:
I'm not sure why this isn't an xref. I'm also not sure why this content isn't in 4.17. Wanted to point out this discrepancy.

Reply:
You are right. This was intended to be added as an xref. I put the link to the module here for my reference and I missed converting it to an xref.

Updated as per the suggestion.